tests/test_dse.py

```diff
@@ -701,4 +701,4 @@ def test_tti_v2_rewrite_aggressive_opcounts(space_order, expected):
 
 
 if __name__ == "__main__":
-    test_tti_rewrite_aggressive_opcounts(16, 270)
+    test_tti_v2_rewrite_aggressive_opcounts(4, 185)
```

Comment:
leftover?

Reply:
Only changed which function to debug doenst really.matter